src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -237,7 +242,9 @@ private string ExtractGhesBaseUrl(string ghesApiUrl)
             await using (var metadataArchiveContent = _fileSystemProvider.OpenRead(metadataArchiveDownloadFilePath))
 #pragma warning restore IDE0063
             {
-                return _awsApi.HasValue()
+                return useGithubStorage
+                    ? await UploadArchivesToGithub(githubTargetOrg, gitArchiveUploadFileName, gitArchiveContent, metadataArchiveUploadFileName, metadataArchiveContent)
+                    : _awsApi.HasValue()
```

Comment:
nit: same here too, I'd use `if..else` for readability

Reply:
[85190e8](https://github.com/github/gh-gei/pull/1280/commits/85190e80419eb38939609a570ce854445ab51790) 👍🏽